README.md

```diff
@@ -179,47 +169,6 @@ Now we can port this UDF to Opaque as follows:
 
 3. Finally, implement the UDF in C++. In [`FlatbuffersExpressionEvaluator#eval_helper`](src/enclave/Enclave/ExpressionEvaluation.h), add a case for `tuix::ExprUnion_DotProduct`. Within that case, cast the expression to a `tuix::DotProduct`, recursively evaluate the left and right children, perform the dot product computation on them, and construct a `DoubleField` containing the result.
 
-## Launch Token and Remote Attestation
```

Comment:
How does remote attestation work now, and how do the enclaves get the shared data key?

Reply:
Unaddressed.

Reply:
Since Opaque has moved to the OpenEnclave SDK, the remote attestation protocol has also been moved to the new protocol that is based on [DCAP](https://api.portal.trustedservices.intel.com/provisioning-certification). Also see [here](https://github.com/openenclave/openenclave/blob/v0.9.0/samples/remote_attestation/common/attestation.cpp#L77) for some OpenEnclave sample code.